frontend/packages/ux-editor/src/components/config/componentSpecificContent/Summary2/Summary2Component.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import type { IGenericEditComponent } from '../../componentConfig';
+import { ComponentType } from 'app-shared/types/ComponentType';
+import {
+  StudioCard,
+  StudioHeading,
+  StudioNativeSelect,
+  StudioParagraph,
+  StudioTextfield,
+} from '@studio/components';
+import { useFormLayoutsQuery } from '../../../../hooks/queries/useFormLayoutsQuery';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import { useAppContext, useComponentTypeName } from '../../../../hooks';
+import { useTranslation } from 'react-i18next';
+import { Summmary2ComponentTargetSelector } from './Summary2ComponentTargetSelector';
+import { getAllLayoutComponents } from '../../../../utils/formLayoutUtils';
+
+export const Summary2Component = ({
+  component,
+  handleComponentChange,
+}: IGenericEditComponent<ComponentType.Summary2>) => {
+  const { t } = useTranslation();
+  const { org, app } = useStudioEnvironmentParams();
+  const { selectedFormLayoutSetName, selectedFormLayoutName } = useAppContext();
+  const { data: formLayoutsData } = useFormLayoutsQuery(org, app, selectedFormLayoutSetName);
+  const componentTypeName = useComponentTypeName();
+
+  const excludedComponents = [
+    ComponentType.Summary2,
+    ComponentType.NavigationButtons,
+    ComponentType.NavigationBar,
+  ]; // TODO: Add more components that should be excluded
```

Comment:
Is this comment supposed to be there? 😄

Reply:
I added this just to indicate for this issue #13528 that this exclusion list is not complete, and should be considered when doing that task.

Reply:
okey nice 😄 
Maybe you can add a comment in the code with the explanation you gave here as well as adding the url to the issue? 😄

Reply:
Good idea 👍 I pushed a change to the wording